src/core/annotation.js

```diff
@@ -2090,27 +2120,46 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
       this.data.fieldValue = asValue;
     }
 
+    let yes = "Yes";
+    if (this.data.fieldValue !== null && this.data.fieldValue !== "Off") {
+      yes = this.data.fieldValue;
+    }
```

Comment:
Optional nit: Maybe ensure that the `yes`-value cannot accidentally be modified below by doing something like:
```javascript
    const yes =
      this.data.fieldValue !== null && this.data.fieldValue !== "Off"
        ? this.data.fieldValue
        : "Yes";
```